test/layers/validation/leaks_mt.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+#include "fixtures.hpp"
+
+#include <iostream>
+#include <thread>
+#include <vector>
+
+INSTANTIATE_TEST_SUITE_P(threadCountForValDeviceTest, valDeviceTestMultithreaded,
+                         ::testing::Values(8));
```

Comment:
Add to parameter values: 2 threads, and max threads, ie. [hardware_concurrency](https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency)

Reply:
Done.